deploy/mod/deploy_rgw.py

```diff
@@ -58,7 +58,8 @@ def restart_rgw(self):
         index = 0
         for rgw_node in self.cluster['rgw']:
             index_end = index + int(self.cluster['rgw_num'])
-            stdout, stderr = common.pdsh(self.cluster['user'],[rgw_node],'host_name=`hostname -s`;for inst in {%s..%s}; do radosgw -n client.radosgw.${host_name}-$inst; done;' % (self.cluster['rgw_index'][index],self.cluster['rgw_index'][index_end-1]), option="console|check_return")
+            stdout, stderr = common.pdsh(self.cluster['user'],[rgw_node],'host_name=`hostname -s`;for inst in {%s..%s}; do radosgw -n client.radosgw.${host_name}-$inst -c /etc/ceph/ceph.conf; done;' % (self.cluster['rgw_index'][index],self.cluster['rgw_index'][index_end-1]), option="console|check_return")
+			index = index_end
```

Comment:
From the man radosgw:  -c ceph.conf, --conf=ceph.conf
 Use ceph.conf configuration file instead of the default /etc/ceph/ceph.conf to determine monitor addresses during startup. Why add -c /etc/ceph/ceph.conf? radosgw can't find this ?

Reply:
Oh,  that's redundant. I think clearer instructions are better.
The important point is next line.

Reply:
ok, i missed the next line.  I think we can remove -c /etc/ceph/ceph.conf. And others looks good.

Reply:
ok, I will remove -c /etc/ceph/ceph.conf.